src/tests/unit/gna/ngraph/transformations/gna_pwl_LP_transformation.cpp

```diff
@@ -0,0 +1,238 @@
+        auto b = ngraph::builder::makeConstant(constPrecision, ov::Shape{3}, testValues.expected.b);
+        auto knots = ngraph::builder::makeConstant(constPrecision, ov::Shape{3}, testValues.expected.knots);
+
+        if (!testValues.expected.dequantizationBefore.empty() || inputPrecision.is_real()) {
+            pwl = std::make_shared<intel_gna::op::Pwl>(dequantizationOpBefore, m, b, knots);
+        } else {
+            pwl = std::make_shared<ngraph::op::TypeRelaxed<intel_gna::op::Pwl>>(
+                std::vector<element::Type>{ element::f32, element::f32, element::f32, element::f32 },
+                std::vector<element::Type>{ element::f32 },
+                ngraph::op::TemporaryReplaceOutputType(dequantizationOpBefore, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(m, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(b, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(knots, element::f32).get());
+            as_type_ptr<ngraph::op::TypeRelaxed<intel_gna::op::Pwl>>(pwl)->set_overridden_output_type(inputPrecision);
+        }
+        pwl->set_friendly_name("output");
+
+        auto result = std::make_shared<ngraph::opset8::Result>(pwl);
+        ref_func_ = std::make_shared<ngraph::Function>(
+                ngraph::ResultVector{result},
+                ngraph::ParameterVector{input},
+                "PWLTransformation");
+    }
+#include "ngraph/pass/visualize_tree.hpp"
+    ngraph::pass::VisualizeTree("/home/vzinovie/work/model_dumps/reference.dot").run_on_model(ref_func_);
+}
+
+void PwlLPTransformationTest::Validate() {
+    ngraph::pass::Manager m;
+    m.register_pass<ngraph::pass::InitNodeInfo>();
+    m.register_pass<ngraph::pass::low_precision::PWLTransformation>(
+        ngraph::pass::low_precision::LayerTransformation::Params(
+            true,
+            element::f32,
+            ngraph::pass::low_precision::precision_set::int8_int16_int32_support));
+    m.run_passes(func_);
+#include "ngraph/pass/visualize_tree.hpp"
+    ngraph::pass::VisualizeTree("/home/vzinovie/work/model_dumps/transformed.dot").run_on_model(func_);
+    auto res = compare_functions(func_, ref_func_, true, true);
+    ASSERT_TRUE(res.first) << res.second;
+}
+
+std::vector<Shape> inputShapes = {
+    {1, 3}
+};
+
+element::TypeVector netPrecisions = {
+    element::f16,
+    element::f32
+};
+
+element::TypeVector inputPrecisions = {
+    element::i8, element::u8,
+    element::i16, element::u16,
+    element::i32, element::u32,
+};
+
+std::vector<PwlLPTransformationTestValues> testValues = {
+    {
+        {
+            {{element::f32}, {0.5f}, {2.f}},
+            {1.f, 2.f, 4.f},
+            {4.f, 2.f, 1.f},
+            {-8.f, 0.5f, 2.f}
+        },
+        {
+            {},
+            {2.f, 4.f, 8.f},
+            {3.f, 0.f, -3.f},
+            {-3.5f, 0.75f, 1.5f},
+            true
+        },
+    },
+};
```

Comment:
please, add test cases when PWL transformation has to be ignored: per-channel quantization, empty dequantization...